### ISSUE TITLE
DatePicker: use input number as year navigator

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#8950]
### Description

When you use the date picker to select a date of birth, you may need to navigate to years ago. With the current selector you either need to include a lot of years, or you need to navigate multiple times.

### Describe the solution you would like

Instead of a select input it would be better (IMO) to have a numeric input.
```html
<input type="number" min="..." max="..." step="1" value="..."/>
```
This allows you to spin up and down, but also to simply enter a year.

**Comment by melloware** [comment]
If you look in the 0-datepicker.js thre is a bunch of comments like this...

```xml
//type="number" min="' + minHour + '" max="' + maxHour + '" - does not work well on Firefox 70, so we donÂ´t use it
```

FF 70 was a while ago but all the DatePicker inputs couldn't probably be changed to browser numbers.

**Comment by jepsar** [comment]
I'll leave this issue open for a second look once the components have received their accessibility update.

**Comment by melloware** [comment]
I have made all the accessibility updates. The difference seems to be that the other libs have this year picker and decade picker which we do not. It looks like it would be a lot of work to port and test.
